pom.xml

```diff
@@ -436,6 +436,11 @@
                             <artifactId>lucene-queryparser</artifactId>
                             <version>5.5.4</version>
                         </exlude>
+                        <exlude>
```

Comment:
There's no Trello link fo this new audit exclusion and it appears to be related to the one above which is actually unrelated.